### DISCUSSION TITLE
Trakt.tv Integration

### DISCUSSION THREAD

**Discussion by Gvex95** [seerr-team/seerr#239] [Ideas]
Hi
First, thanks for this amazing piece of software. Only thing that this is missing, at least for my needs is integration with Trakt.tv.
It would be great that each user in it's settings can authenticate with trakt, and then choose list which will be displayed on the main page. For example i have trakt sync tool setup with plex, and i see my watchlist from trakt -> plex -> jellyseer (only issue is that i don't  see all the titles, so this is maybe existing bug).
But overall with lists from trakt (maybe not only user list, but ability to see custom list by providing url in settings), it will be really nice workflow, where i could use my mobile add some movies to some custom list, and when i get home just request them without needing to search them. Another idea that pop in my mind just now is bulk request, that is request all items from some list (that would save a lot of time).

I hope you will consider my idea and hopefully implement it, it would be really amazing. Thanks :)

**Comment by fallenbagel** [comment]
Do you mean like when you add a media to a list from trakt, it adds it to jellyseerr? 🤔

As for not being able to see some, that could be due to the naming between tmdb and trakt being different. Afaik some of the trakt shows are still tvdb not tmdb

**Comment by Gvex95** [comment]
Yes, general trakt integration. So the flow would be:
1. I authenticate my user with trakt
2. I get additional trakt settings, where i can choose my lists that i want to show in the UI of the jellyseer
3. Then i can request a specific movie from that list, or maybe bulk option to request all movies from the list with specific profile.
I am not sure how hard is this to implement but it would really be a cool feature, and make our life a lot easier :)

**Comment by umfk** [comment]
I think this would be a fantastic addition to Jellyseerr. A lot of other tools have integrations with trakt and this way Jellyseerr would effectively get integrations with those other tools for free.

For example I'm currently trying to get personalized recommendations into my media stack and one tool for that would be couchmoney.tv which takes a users trakt ratings and creates new trakt lists with recommendations. If I could automatically see these lists in Jellyseerr I could then choose which of the recommended movies/series I want to request.

Any end user (my wife e.g.) would then only need Jellyseerr and Jellyfin, everything else would run automatically in the background.

**Comment by BostonBella** [comment]
This would be amazing, the option to add TMDB list as sliders is already available so I hope we'll be able to add Trakt list sliders at some point as well. I think anyone serious about movie/show lists uses Trakt because it integrates with so many other things.

**Comment by andrew-manger** [comment]
@Fallenbagel Also just want to bump this one! An integration with Trakt.tv and/or Letterboxd would be great! It would be even better if Jellyseerr could perform these functions natively, but if that's not in the cards, I may be able to volunteer to draw up some API calls for the integration

**Comment by pejotigrek** [comment]
I'm gonna dig this old-ish thread to add, that trakt.tv integration, with the possibility to not only see OWN ratings on the main dashboard page, but also ability to rate movies and sync those ratings with trakt account would be a game-changer!
